---
title:  'Juliana assis'
visible: false
---
    

![Juliana Assis](/assets/img/staff/juliana-assis.png)

###  Juliana Assis

_genomics, transcriptomics, system biology, metabolomics, data integration_

_email_|  juliana.assis@nbis.se  
---|---  
![](/assets/img/orcid_24x24_bw.png)| <https://orcid.org/0000-0001-5995-8684>  
  


I am a Biologist with a master and doctor degree in bioinformatics, with extensive experience in Whole Genome Sequencing (short and long reads). I have been working with bioinformatics since 2009, starting in my undergraduate studies. My field lies broadly in the multi-omics data including Genomic, Transcriptomic, Metagenomic and currently data Integration.

I got my masters and Ph.D. in Bioinformatics at the Universidade Federal de Minas Gerais (UFMG, Brazil), with a collaborative period in the University of Georgia (UGA, USA), and at the Wellcome Sanger Institute, Sanger - UK.

The group where I developed my master’s and Ph.D. is a Platform of Bioinformatic which is a government agency under the Ministry of Health in Brazil(Oswaldo Cruz Foundation). The platform is responsible to help others researches to design the experiments and perform the bioinformatics analysis, as also to provide the infrastructure for the analysis. At this platform I was working with NGS (short and long reads) data processing,including genomic, RNA-seq, Hi-C, WGS, metagenome, variant calling. The group where I worked for 2 years in the USA, is responsible to develop the VEupathDB “Database Family”, which gave me the opportunity to learn the database infrastructure and the data integration. I also did a short training (a month) in the Wellcome Sanger Institute (Sanger,UK) to learn how to improve the quality of eukaryotic genome annotation.

In 2019, I moved to Copenhagen - Denmark. During my Postdoctoral Researcher at the Novo Nordisk Foundation Center for Basic Metabolic Research - CBMR, I worked in the Human microbiome field, with long reads - Nanopore, PacBio ISO-seq data and short reads (illumina) to perform metagenome assembly (binning methods) and integrating with the other omics-data (metatranscriptomics and metametabolomics). In addition, I was also involved in some 16S rRNA taxonomic studies. During the last couple of years, I have published papers involving the high quality of genomics studies.

###  Selected publications 

Andrade, L. F. de, Mourao, M. de M., Geraldo, J. A., Coelho, F. S., Silva, L. L., Neves, R. H., Volpini, A., Machado-Silva, J. R., Araujo, N., Nacif-Pimenta, R., and others (2014) Regulation of Schistosoma mansoni development and reproduction by the mitogen-activated protein kinase signaling pathway. _PLoS neglected tropical diseases_ , **8** , e2949.  
  
Pylro, V. S., Oliveira, F. S., Morais, D. K., Cuadros-Orellana, S., Pais, F. S.-M., Medeiros, J. D., Geraldo, J. A., Gilbert, J., Volpini, A. C., and Fernandes, G. R. (2016) ZIKV–CDB: a collaborative database to guide research linking SncRNAs and ZIKA virus disease symptoms. _PLoS neglected tropical diseases_ , **10** , e0004817.  
  
Chien, J.-T., Pakala, S. B., Geraldo, J. A., Lapp, S. A., Humphrey, J. C., Barnwell, J. W., Kissinger, J. C., and Galinski, M. R. (2016) High-quality genome assembly and annotation for Plasmodium coatneyi, generated using single-molecule real-time PacBio technology. _Genome announcements_ , **4** , e00883–16.  
  
Adema, C. M., Hillier, L. D. W., Jones, C. S., Loker, E. S., Knight, M., Minx, P., Oliveira, G., Raghavan, N., Shedlock, A., Do Amaral, L. R., and others (2017) Whole genome analysis of a schistosomiasis-transmitting freshwater snail. _Nature communications_ , **8** , 1–12.  
  
Maldonado, L. L., Assis, J., Araújo, F. M. G., Salim, A. C. M., Macchiaroli, N., Cucher, M., Camicia, F., Fox, A., Rosenzvit, M., Oliveira, G., and others (2017) The Echinococcus canadensis (G7) genome: a key knowledge of parasitic platyhelminth human diseases. _Bmc Genomics_ , **18** , 1–23.  
  
Rosse, I. C., Assis, J. G., Oliveira, F. S., Leite, L. R., Araujo, F., Zerlotini, A., Volpini, A., Dominitini, A. J., Lopes, B. C., Arbex, W. A., and others (2017) Whole genome sequencing of Guzerá cattle reveals genetic variants in candidate genes for production, disease resistance, and heat tolerance. _Mammalian genome_ , **28** , 66–80.  
  
Lapp, S. A., Geraldo, J. A., Chien, J.-T., Ay, F., Pakala, S. B., Batugedara, G., Humphrey, J., DeBARRY, J. D., Le Roch, K. G., Galinski, M. R., and others (2018) PacBio assembly of a Plasmodium knowlesi genome sequence with Hi-C correction and manual annotation of the SICAvar gene family. _Parasitology_ , **145** , 71–84.  
  
Figueiró, H. V., Li, G., Trindade, F. J., Assis, J., Pais, F., Fernandes, G., Santos, S. H. D., Hughes, G. M., Komissarov, A., Antunes, A., and others (2017) Genome-wide signatures of complex introgression and adaptive evolution in the big cats. _Science Advances_ , **3** , e1700299.  
  
Geraldo, J. A., Silva, F., Horácio, E. C. A., Leão, A., Brayan, A., Rocha Fernandes, G. da, and Coimbra, N. D. R. (2018) Highlights of the 2nd Brazilian student council symposium, organized by ISCB Regional Student Group Brazil. _PeerJ PrePrints_.  
  
Brito Raı́ssa N, Geraldo, J. A., Monteiro, F. A., Lazoski, C., Souza, R. C. M., and Abad-Franch, F. (2019) Transcriptome-based molecular systematics: Rhodnius montenegrensis (Triatominae) and its position within the Rhodnius prolixus–Rhodnius robustus cryptic–species complex. _Parasites & vectors_, **12** , 1–16.  
  
Martinez-Villegas, L., Assis-Geraldo, J., Koerich, L. B., Collier, T. C., Lee, Y., Main, B. J., Rodrigues, N. B., Orfano, A. S., Pires, A. C. A. M., Campolina, T. B., and others (2019) Characterization of the complete mitogenome of Anopheles aquasalis, and phylogenetic divergences among Anopheles from diverse geographic zones. _PloS one_ , **14** , e0219523.  
  
Shome, S., Parra, R. G., Fatima, N., Monzon, A. M., Cuypers, B., Moosa, Y., Coimbra, N. D. R., Assis, J., Giner-Delgado, C., Dönertaş, H. M., and others (2019) Global network of computational biology communities: ISCB’s Regional Student Groups breaking barriers. _F1000Research_ , **8**.  
  
Airey, E., Portelli, S., Xavier, J. S., Myung, Y. C., Silk, M., Karmakar, M., Velloso, J. P. L., Rodrigues, C. H. M., Parate, H. H., Garg, A., and others (2021) Identifying Genotype–Phenotype Correlations via Integrative Mutation Analysis. In, _Artificial Neural Networks_. Humana, New York, NY, pp. 1–32.  
  
Gnanasekaran, T., Assis Geraldo, J., Ahrenkiel, D. W., Alvarez-Silva, C., Saenz, C., Khan, A., Hanteer, O., Gunalan, V., Trost, K., Moritz, T., and others (2021) Ecological Adaptation and Succession of Human Fecal Microbial Communities in an Automated In Vitro Fermentation System. _Msystems_ , **6** , e00232–21. 

![Juliana Assis](/assets/img/staff/juliana-assis.png)

###  Juliana Assis

_genomics, transcriptomics, system biology, metabolomics, data integration_

_email_|  juliana.assis@nbis.se  
---|---  
![](/assets/img/orcid_24x24_bw.png)| <https://orcid.org/0000-0001-5995-8684>  
  


I am a Biologist with a master and doctor degree in bioinformatics, with extensive experience in Whole Genome Sequencing (short and long reads). I have been working with bioinformatics since 2009, starting in my undergraduate studies. My field lies broadly in the multi-omics data including Genomic, Transcriptomic, Metagenomic and currently data Integration.

I got my masters and Ph.D. in Bioinformatics at the Universidade Federal de Minas Gerais (UFMG, Brazil), with a collaborative period in the University of Georgia (UGA, USA), and at the Wellcome Sanger Institute, Sanger - UK.

The group where I developed my master’s and Ph.D. is a Platform of Bioinformatic which is a government agency under the Ministry of Health in Brazil(Oswaldo Cruz Foundation). The platform is responsible to help others researches to design the experiments and perform the bioinformatics analysis, as also to provide the infrastructure for the analysis. At this platform I was working with NGS (short and long reads) data processing,including genomic, RNA-seq, Hi-C, WGS, metagenome, variant calling. The group where I worked for 2 years in the USA, is responsible to develop the VEupathDB “Database Family”, which gave me the opportunity to learn the database infrastructure and the data integration. I also did a short training (a month) in the Wellcome Sanger Institute (Sanger,UK) to learn how to improve the quality of eukaryotic genome annotation.

In 2019, I moved to Copenhagen - Denmark. During my Postdoctoral Researcher at the Novo Nordisk Foundation Center for Basic Metabolic Research - CBMR, I worked in the Human microbiome field, with long reads - Nanopore, PacBio ISO-seq data and short reads (illumina) to perform metagenome assembly (binning methods) and integrating with the other omics-data (metatranscriptomics and metametabolomics). In addition, I was also involved in some 16S rRNA taxonomic studies. During the last couple of years, I have published papers involving the high quality of genomics studies.

###  Selected publications 

Andrade, L. F. de, Mourao, M. de M., Geraldo, J. A., Coelho, F. S., Silva, L. L., Neves, R. H., Volpini, A., Machado-Silva, J. R., Araujo, N., Nacif-Pimenta, R., and others (2014) Regulation of Schistosoma mansoni development and reproduction by the mitogen-activated protein kinase signaling pathway. _PLoS neglected tropical diseases_ , **8** , e2949.  
  
Pylro, V. S., Oliveira, F. S., Morais, D. K., Cuadros-Orellana, S., Pais, F. S.-M., Medeiros, J. D., Geraldo, J. A., Gilbert, J., Volpini, A. C., and Fernandes, G. R. (2016) ZIKV–CDB: a collaborative database to guide research linking SncRNAs and ZIKA virus disease symptoms. _PLoS neglected tropical diseases_ , **10** , e0004817.  
  
Chien, J.-T., Pakala, S. B., Geraldo, J. A., Lapp, S. A., Humphrey, J. C., Barnwell, J. W., Kissinger, J. C., and Galinski, M. R. (2016) High-quality genome assembly and annotation for Plasmodium coatneyi, generated using single-molecule real-time PacBio technology. _Genome announcements_ , **4** , e00883–16.  
  
Adema, C. M., Hillier, L. D. W., Jones, C. S., Loker, E. S., Knight, M., Minx, P., Oliveira, G., Raghavan, N., Shedlock, A., Do Amaral, L. R., and others (2017) Whole genome analysis of a schistosomiasis-transmitting freshwater snail. _Nature communications_ , **8** , 1–12.  
  
Maldonado, L. L., Assis, J., Araújo, F. M. G., Salim, A. C. M., Macchiaroli, N., Cucher, M., Camicia, F., Fox, A., Rosenzvit, M., Oliveira, G., and others (2017) The Echinococcus canadensis (G7) genome: a key knowledge of parasitic platyhelminth human diseases. _Bmc Genomics_ , **18** , 1–23.  
  
Rosse, I. C., Assis, J. G., Oliveira, F. S., Leite, L. R., Araujo, F., Zerlotini, A., Volpini, A., Dominitini, A. J., Lopes, B. C., Arbex, W. A., and others (2017) Whole genome sequencing of Guzerá cattle reveals genetic variants in candidate genes for production, disease resistance, and heat tolerance. _Mammalian genome_ , **28** , 66–80.  
  
Lapp, S. A., Geraldo, J. A., Chien, J.-T., Ay, F., Pakala, S. B., Batugedara, G., Humphrey, J., DeBARRY, J. D., Le Roch, K. G., Galinski, M. R., and others (2018) PacBio assembly of a Plasmodium knowlesi genome sequence with Hi-C correction and manual annotation of the SICAvar gene family. _Parasitology_ , **145** , 71–84.  
  
Figueiró, H. V., Li, G., Trindade, F. J., Assis, J., Pais, F., Fernandes, G., Santos, S. H. D., Hughes, G. M., Komissarov, A., Antunes, A., and others (2017) Genome-wide signatures of complex introgression and adaptive evolution in the big cats. _Science Advances_ , **3** , e1700299.  
  
Geraldo, J. A., Silva, F., Horácio, E. C. A., Leão, A., Brayan, A., Rocha Fernandes, G. da, and Coimbra, N. D. R. (2018) Highlights of the 2nd Brazilian student council symposium, organized by ISCB Regional Student Group Brazil. _PeerJ PrePrints_.  
  
Brito Raı́ssa N, Geraldo, J. A., Monteiro, F. A., Lazoski, C., Souza, R. C. M., and Abad-Franch, F. (2019) Transcriptome-based molecular systematics: Rhodnius montenegrensis (Triatominae) and its position within the Rhodnius prolixus–Rhodnius robustus cryptic–species complex. _Parasites & vectors_, **12** , 1–16.  
  
Martinez-Villegas, L., Assis-Geraldo, J., Koerich, L. B., Collier, T. C., Lee, Y., Main, B. J., Rodrigues, N. B., Orfano, A. S., Pires, A. C. A. M., Campolina, T. B., and others (2019) Characterization of the complete mitogenome of Anopheles aquasalis, and phylogenetic divergences among Anopheles from diverse geographic zones. _PloS one_ , **14** , e0219523.  
  
Shome, S., Parra, R. G., Fatima, N., Monzon, A. M., Cuypers, B., Moosa, Y., Coimbra, N. D. R., Assis, J., Giner-Delgado, C., Dönertaş, H. M., and others (2019) Global network of computational biology communities: ISCB’s Regional Student Groups breaking barriers. _F1000Research_ , **8**.  
  
Airey, E., Portelli, S., Xavier, J. S., Myung, Y. C., Silk, M., Karmakar, M., Velloso, J. P. L., Rodrigues, C. H. M., Parate, H. H., Garg, A., and others (2021) Identifying Genotype–Phenotype Correlations via Integrative Mutation Analysis. In, _Artificial Neural Networks_. Humana, New York, NY, pp. 1–32.  
  
Gnanasekaran, T., Assis Geraldo, J., Ahrenkiel, D. W., Alvarez-Silva, C., Saenz, C., Khan, A., Hanteer, O., Gunalan, V., Trost, K., Moritz, T., and others (2021) Ecological Adaptation and Succession of Human Fecal Microbial Communities in an Automated In Vitro Fermentation System. _Msystems_ , **6** , e00232–21. 
