---
title:  'Lucile soler nbis expert'
visible: false
---
    

![Lucile Soler](/assets/img/staff/lucile-soler.png)

###  Lucile Soler NBIS expert

_genome annotation, whole genome alignment, comparative genomic_

_email_|  lucile.soler@nbis.se  
---|---  
_phone_|  +46 (0)7 20 17 33 10  
  


After a master degree in bioinformatics at the university of Montpellier in France, I did a PhD in Bioinformatics in the aquaculture team at the CIRAD (Agricultural research for development) in Montpellier, France. I worked on the analysis of linkage groups associated to the sex cascade in the Nile tilapia (Oreochromis niloticus). I collected and integrated heterogeneous information (genomic, protein, mapping information, …) existing on tilapia using comparative genomic analyses and BioPerl scripts. Then, when the genome sequence was available, I assembled scaffolds into chromosomes and I annotated, analysed the chromosome of interest using methods, tools and data I developed and already existing tools (such as maker). I finally analysed genes of interest using phylogeny analyses and synteny groups. This work was done using the high-performance computing Unix/Linux environment of the CIRAD team of bioinformatics.

Then I worked for 18 months at Syngenta in Toulouse (France), where I developed a pipeline to transfer markers coming from NGS sequencing. I tested and used whole genome alignment tool such as Satsuma, Mummer and Lastz and in-house perl scripts to create this pipeline. I am working as an annotation expert here at NBIS.

![Lucile Soler](/assets/img/staff/lucile-soler.png)

###  Lucile Soler NBIS expert

_genome annotation, whole genome alignment, comparative genomic_

_email_|  lucile.soler@nbis.se  
---|---  
_phone_|  +46 (0)7 20 17 33 10  
  


After a master degree in bioinformatics at the university of Montpellier in France, I did a PhD in Bioinformatics in the aquaculture team at the CIRAD (Agricultural research for development) in Montpellier, France. I worked on the analysis of linkage groups associated to the sex cascade in the Nile tilapia (Oreochromis niloticus). I collected and integrated heterogeneous information (genomic, protein, mapping information, …) existing on tilapia using comparative genomic analyses and BioPerl scripts. Then, when the genome sequence was available, I assembled scaffolds into chromosomes and I annotated, analysed the chromosome of interest using methods, tools and data I developed and already existing tools (such as maker). I finally analysed genes of interest using phylogeny analyses and synteny groups. This work was done using the high-performance computing Unix/Linux environment of the CIRAD team of bioinformatics.

Then I worked for 18 months at Syngenta in Toulouse (France), where I developed a pipeline to transfer markers coming from NGS sequencing. I tested and used whole genome alignment tool such as Satsuma, Mummer and Lastz and in-house perl scripts to create this pipeline. I am working as an annotation expert here at NBIS.
