---
title:  'Mahesh binzer-panchal nbis expert'
visible: false
---
    

![Mahesh Binzer-Panchal](/assets/img/staff/mahesh-panchal.jpg)

###  Mahesh Binzer-Panchal NBIS expert

_genome assembly, transcriptome assembly, metagenomic assembly, variant calling, amplicon analysis, Nextflow, shell scripting, perl, groovy, java, R_

_email_|  mahesh.binzer-panchal@nbis.se  
---|---  
  


I have a background in Computer Science and Mathematics from Imperial College London. I then went on do to my PhD on the Automation of Nested Clade Phylogeographic Analysis at Reading University. I then pursued postdoctoral research at the Max Planck Institute in Plön, Germany, working on Bioinformatics related to lake-river adaptation and host-parasite coevolution in three-spined stickleback. My research included working with various NGS data, doing de novo assembly, resequencing studies, transcriptomic studies, and metagenomic studies.

I currently specialise in de novo genome assembly in the NBIS genomic assembly group, from quality control to filtering and assembly, and then assembly validation. I also specialise in writing and supporting Nextflow workflows, and teaching both these subject areas.

###  Selected publications 

Del Angel, V. D., Hjerde, E., Sterck, L., Capella-Gutierrez, S., Notredame, C., Pettersson, O. V., Amselem, J., Bouri, L., Bocs, S., Klopp, C., and others (2018) Ten steps to get started in Genome Assembly and Annotation. _F1000Research_ , **7**.  
  
Panchal, M. and Beaumont, M. A. (2007) The automation and evaluation of nested clade phylogeographic analysis. _Evolution_ , **61** , 1466–1480.  
  
Panchal, M. (2007) The automation of nested clade phylogeographic analysis. _Bioinformatics_ , **23** , 509–510.  
  
Beaumont, M. A., Nielsen, R., Robert, C., Hey, J., Gaggiotti, O., Knowles, L., Estoup, A., Panchal, M., Corander, J., Hickerson, M., and others (2010) In defence of model-based inference in phylogeography. _Molecular Ecology_ , **19** , 436–446.  
  
Panchal, M. and Beaumont, M. A. (2010) Evaluating nested clade phylogeographic analysis under models of restricted gene flow. _Systematic Biology_ , syq022.  
  
Beaumont, M. A. and Panchal, M. (2008) On the validity of nested clade phylogeographical analysis. _Molecular Ecology_ , **17** , 2563–2565.  
  
Feulner, P. G. D., Chain, F. J. J., Panchal, M., Eizaguirre, C., Kalbe, M., Lenz, T. L., Mundry, M., Samonte, I. E., Stoll, M., Milinski, M., and others (2013) Genome-wide patterns of standing genetic variation in a marine population of three-spined sticklebacks. _Molecular ecology_ , **22** , 635–649.  
  
Chain, F. J. J., Feulner, P. G. D., Panchal, M., Eizaguirre, C., Samonte, I. E., Kalbe, M., Lenz, T. L., Stoll, M., Bornberg-Bauer, E., Milinski, M., and others (2014) Extensive copy-number variation of young genes across stickleback populations. _PLoS genetics_ , **10** , e1004830.  
  
Feulner, P. G. D., Chain, F. J. J., Panchal, M., Huang, Y., Eizaguirre, C., Kalbe, M., Lenz, T. L., Samonte, I. E., Stoll, M., Bornberg-Bauer, E., and others (2015) Genomics of divergence along a continuum of parapatric population differentiation. _PLoS Genet_ , **11** , e1004966. 

![Mahesh Binzer-Panchal](/assets/img/staff/mahesh-panchal.jpg)

###  Mahesh Binzer-Panchal NBIS expert

_genome assembly, transcriptome assembly, metagenomic assembly, variant calling, amplicon analysis, Nextflow, shell scripting, perl, groovy, java, R_

_email_|  mahesh.binzer-panchal@nbis.se  
---|---  
  


I have a background in Computer Science and Mathematics from Imperial College London. I then went on do to my PhD on the Automation of Nested Clade Phylogeographic Analysis at Reading University. I then pursued postdoctoral research at the Max Planck Institute in Plön, Germany, working on Bioinformatics related to lake-river adaptation and host-parasite coevolution in three-spined stickleback. My research included working with various NGS data, doing de novo assembly, resequencing studies, transcriptomic studies, and metagenomic studies.

I currently specialise in de novo genome assembly in the NBIS genomic assembly group, from quality control to filtering and assembly, and then assembly validation. I also specialise in writing and supporting Nextflow workflows, and teaching both these subject areas.

###  Selected publications 

Del Angel, V. D., Hjerde, E., Sterck, L., Capella-Gutierrez, S., Notredame, C., Pettersson, O. V., Amselem, J., Bouri, L., Bocs, S., Klopp, C., and others (2018) Ten steps to get started in Genome Assembly and Annotation. _F1000Research_ , **7**.  
  
Panchal, M. and Beaumont, M. A. (2007) The automation and evaluation of nested clade phylogeographic analysis. _Evolution_ , **61** , 1466–1480.  
  
Panchal, M. (2007) The automation of nested clade phylogeographic analysis. _Bioinformatics_ , **23** , 509–510.  
  
Beaumont, M. A., Nielsen, R., Robert, C., Hey, J., Gaggiotti, O., Knowles, L., Estoup, A., Panchal, M., Corander, J., Hickerson, M., and others (2010) In defence of model-based inference in phylogeography. _Molecular Ecology_ , **19** , 436–446.  
  
Panchal, M. and Beaumont, M. A. (2010) Evaluating nested clade phylogeographic analysis under models of restricted gene flow. _Systematic Biology_ , syq022.  
  
Beaumont, M. A. and Panchal, M. (2008) On the validity of nested clade phylogeographical analysis. _Molecular Ecology_ , **17** , 2563–2565.  
  
Feulner, P. G. D., Chain, F. J. J., Panchal, M., Eizaguirre, C., Kalbe, M., Lenz, T. L., Mundry, M., Samonte, I. E., Stoll, M., Milinski, M., and others (2013) Genome-wide patterns of standing genetic variation in a marine population of three-spined sticklebacks. _Molecular ecology_ , **22** , 635–649.  
  
Chain, F. J. J., Feulner, P. G. D., Panchal, M., Eizaguirre, C., Samonte, I. E., Kalbe, M., Lenz, T. L., Stoll, M., Bornberg-Bauer, E., Milinski, M., and others (2014) Extensive copy-number variation of young genes across stickleback populations. _PLoS genetics_ , **10** , e1004830.  
  
Feulner, P. G. D., Chain, F. J. J., Panchal, M., Huang, Y., Eizaguirre, C., Kalbe, M., Lenz, T. L., Samonte, I. E., Stoll, M., Bornberg-Bauer, E., and others (2015) Genomics of divergence along a continuum of parapatric population differentiation. _PLoS Genet_ , **11** , e1004966. 
