---
title:  'Long-term support projects'
visible: true
---
    

### Long-term support projects

  * Active projects
  * Past projects



Click on a project for further details and links to publications, analysis tools and data.

_unfold_more_ Show all details _unfold_less_ Hide all details

  * 1000 ancient genomes  PI: Anders Götherström (SU) 

Application round: May 2019 

Subject: Evolutionary Biology/Genetics

  
Co-applicants: Love Dalén (NRM) 

  * A new diagnostic test for rheumatoid arthritis  PI: Rikard Holmdahl (KI) 

Application round: Oct 2018 

Subject: Molecular Medicine & Biomarkers

  
Co-applicants: Inger Gjertsson (GU) 

  * Acquisition of lymphatic endothelial cell fate during zebrafish development  PI: Kaska Koltowska (UU) 

Application round: Feb 2019 

Subject: Cell and Developmental Biology

  
Co-applicants: Hannah Stanforth (UU) 

  * Adaptive deregulation of miRNA genes by genomic rearrangements  PI: Carlos Rovira (LU) 

Application round: Feb 2019 

Subject: Cancer Biology

  * B cell receptor diversity and convergence after COVID-19 vaccination  PI: Karin Loré (KI) 

Application round: Jan 2021 

Subject: Molecular Medicine & Biomarkers

  * Baltic sea CRISPR dynamics  PI: Anders Andersson (KTH) 

Application round: Feb 2016 

Subject: Biodiversity

  * Barcode-linked reads technology for genome sequencing and haplotyping  PI: Afshin Ahmadian (KTH) 

Application round: Feb 2019 

Subject: New Technology Development

  * Cellular heterogeneity and differentiation during salamander regeneration  PI: Andras Simon (KI) 

Application round: Jan 2020 

Subject: Cell and Developmental Biology

  * Chromatin dynamics and inheritance  PI: Simon Elsässer (KI) 

Application round: May 2017 

Subject: Cell and Developmental Biology

  * Chromatin-associated RNAs and their roles in chromatin regulation  PI: Neus Visa (SU) 

Application round: Feb 2019 

Subject: Cell and Developmental Biology

  
Co-applicants: Jordi Planells (SU), Vincent Pelechano (KI) 

  * Clonal evolution from diagnosis to relapse in acute lymphoblastic leukemia  PI: Ann-Christine Syvänen (UU) 

Application round: Oct 2016 

Subject: Cancer Biology

  * Correlation between patient-derived CAR-T cells and treatment outcome  PI: Magnus Essand (UU) 

Application round: Jan 2021 

Subject: Molecular Medicine & Biomarkers

  * Data integration for molecular classification of multiple sclerosis patients  PI: Maja Jagodic (KI) 

Application round: Feb 2019 

Subject: Molecular Medicine & Biomarkers

Publications:

[ Zheleznyakova _et al._ **Small noncoding RNA profiling across cellular and biofluid compartments and their implications for multiple sclerosis immunopathology.** _Proc Natl Acad Sci USA_ 118:e2011574118 (2021) ](<http://dx.doi.org/10.1073/pnas.2011574118>)

Data: [ncRNA counts](<https://doi.org/10.5878/c1mq-9r62>)

  * Decoding the birth of hair follicles by single-cell RNA-seq  PI: Maria Kasper (KI) 

Application round: Sep 2017 

Subject: Cell and Developmental Biology

  * Defining cell heterogeneity and therapy resistance in neuroblastoma  PI: Per Kogner (KI) 

Application round: Oct 2018 

Subject: Cancer Biology

  
Co-applicants: Ninib Baryawno (KI) 

  * Delineating the genetics in severe scoliosis  PI: Paul Gerdhem (KI) 

Application round: Oct 2020 

Subject: Human Genetics

  
Co-applicants: Elisabet Einarsdottir (KTH), Juha Kere (KI) 

  * Dissection of cell-fate decisions in early B lymphocyte activation  PI: Taras Kreslavskiy (KI) 

Application round: Oct 2019 

Subject: Cell and Developmental Biology

  * Environmental genomics in the Northern krill  PI: Andreas Wallberg (UU) 

Application round: Oct 2018 

Subject: Evolutionary Biology/Genetics

  * Epigenetic regulation of human skin wound healing  PI: Ning Xu Landén (KI) 

Application round: Oct 2020 

Subject: Molecular Medicine & Biomarkers

  * Epigenetics of acute myeloid leukemia  PI: Karl Ekwall (KI) 

Application round: Nov 2014 

Subject: Cancer Biology

  * Establishment of epigenomic landscapes in the Drosophila embryo  PI: Mattias Mannervik (SU) 

Application round: May 2019 

Subject: Cell and Developmental Biology

  * Evolution of human skeletal muscle metabolism  PI: Ola Hansson (LU) 

Application round: Sep 2017 

Subject: Evolutionary Biology/Genetics

  * Evolution of sex chromosomes  PI: Bengt Hansson (LU) 

Application round: Oct 2016 

Subject: Evolutionary Biology/Genetics

  * Evolutionary genomics of Darwin&aposs finches  PI: Leif Andersson (UU) 

Application round: Feb 2019 

Subject: Evolutionary Biology/Genetics

  
Co-applicants: Mats Pettersson (UU) 

  * Evolutionary genomics of the Motacilla (wagtails) radiation  PI: Per Alström (UU) 

Application round: May 2020 

Subject: Evolutionary Biology/Genetics

  
Co-applicants: Erik Enbody (UU), Leif Andersson (UU) 

  * Fat heterogeneity  PI: Kirsty Spalding (KI) 

Application round: Jun 2016 

Subject: Cell and Developmental Biology

  * Fate mapping of human cells  PI: Jonas Frisén (KI) 

Application round: Feb 2016 

Subject: New Technology Development

Publications:

[ Hård _et al._ **Conbase: a software for unsupervised discovery of clonal somatic mutations in single cells through read phasing.** _Genome Biol_ 20:68 (2019) ](<http://dx.doi.org/10.1186/s13059-019-1673-8>)

  * From single nuclei genomes to the arbuscular mycorrhizal tree of life  PI: Anna Rosling Larsson (UU) 

Application round: May 2018 

Subject: Evolutionary Biology/Genetics

  * Genome and metagenome of the deep-sea Swedish sponge Geodia barretti  PI: Paco Cárdenas (UU) 

Application round: Jan 2020 

Subject: Biodiversity

  * Genome-wide consequences of population declines in endangered species  PI: Love Dalén (NRM) 

Application round: Sep 2017 

Subject: Evolutionary Biology/Genetics

  
Co-applicants: Johanna von Seth (SU), Nicolas Dussex (NRM) 

Publications:

[ Larsson _et al._ **Consequences of past climate change and recent human persecution on mitogenomic diversity in the arctic fox.** _Philos Trans R Soc Lond B Biol Sci_ 374:20190212 (2019) ](<http://dx.doi.org/10.1098/rstb.2019.0212>) [ Lord _et al._ **Pre-extinction Demographic Stability and Genomic Signatures of Adaptation in the Woolly Rhinoceros.** _Curr Biol_ 30:3871-3879.e7 (2020) ](<http://dx.doi.org/10.1016/j.cub.2020.07.046>) [ von Seth _et al._ **Genomic insights into the conservation status of the world &aposs last remaining Sumatran rhinoceros populations.** _Nat Commun_ 12:2393 (2021) ](<http://dx.doi.org/10.1038/s41467-021-22386-8>)

  * Genome-wide screen for inhibitors of lysosomal damage-induced cell death  PI: Sonja Aits (LU) 

Application round: Oct 2018 

Subject: Cell and Developmental Biology

  * Genomic analysis of adaptation to climate change in two bumblebee species  PI: Matthew Webster (UU) 

Application round: Sep 2017 

Subject: Evolutionary Biology/Genetics

Publications:

[ Christmas _et al._ **Genetic barriers to historical gene flow between cryptic species of alpine bumblebees revealed by comparative population genomics.** _Mol Biol Evol_ advance online publication (2021) ](<http://dx.doi.org/10.1093/molbev/msab086>)

Data: [WGS reads](<https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP284846>)

  * Genomic analysis of pediatric solid tumors  PI: Monica Nistér (KI) 

Application round: May 2017 

Subject: Cancer Biology

  
Co-applicants: Teresita Diaz de Ståhl (KI), Johanna Sandgren (KI), Maxime Garcia (KI), Gustaf Ljungman (UU) 

Publications:

[ Garcia _et al._ **Sarek: A portable workflow for whole-genome sequencing analysis of germline and somatic variants.** _F1000Res_ 9:63 (2020) ](<http://dx.doi.org/10.12688/f1000research.16665.1>)

Tools and scripts:

[ **Sarek** – Complete open source pipeline to detect germline or somatic variants from normal or tumour/normal whole-genome sequencing data. ](<http://opensource.scilifelab.se/projects/sarek/>)

  * Genomic and transcriptome profiling of MDS cells  PI: Eva Hellström-Lindberg (KI) 

Application round: May 2019 

Subject: Cancer Biology

  
Co-applicants: Seishi Ogawa (Kyoto Univ. and KI) 

  * Genomic dynamics over space and time in salmonid metapopulations  PI: Linda Laikre (SU) 

Application round: May 2018 

Subject: Evolutionary Biology/Genetics

Publications:

[ Kurland _et al._ **Exploring a Pool‐seq‐only approach for gaining population genomic insights in nonmodel species.** _Ecol Evol_ 9:11448-11463 (2019) ](<http://dx.doi.org/10.1002/ece3.5646>)

  * Identification of functionally perturbing non-coding mutations  PI: Robert Måsson (KI) 

Application round: Jan 2017 

Subject: Cancer Biology

  * Impact of spaceflight on mouse heart spatial gene expression  PI: Stefania Giacomello (KTH) 

Application round: May 2020 

Subject: Cell and Developmental Biology

  * Improved gene-finding annotation in complex genomes  PI: Anders Blomberg (GU) 

Application round: May 2019 

Subject: Evolutionary Biology/Genetics

  * Imputing genetic variants in a high kinship human cohort  PI: Åsa Johansson (UU) 

Application round: Jun 2015 

Subject: Human Genetics

Publications:

[ Höglund _et al._ **Improved power and precision with whole genome sequencing data in genome-wide association studies of inflammatory biomarkers.** _Sci Rep_ 9:16844 (2019) ](<http://dx.doi.org/10.1038/s41598-019-53111-7>)

  * Insect aging omics  PI: Christopher Wheat (SU) 

Application round: Jun 2016 

Subject: Evolutionary Biology/Genetics

Publications:

[ Lehmann _et al._ **Metabolome dynamics of diapause in the butterfly Pieris napi: distinguishing maintenance, termination and post-diapause phases.** _J Exp Biol_ 221:jeb169508 (2018) ](<http://dx.doi.org/10.1242/jeb.169508>)

Data: [Cold tolerance and metabolome data](<https://datadryad.org/resource/doi:10.5061/dryad.f0dg5>)

  * Integrate multiple omics in 9000 Swedish individuals  PI: Marju Orho-Melander (LU) 

Application round: Jan 2020 

Subject: Molecular Medicine & Biomarkers

  
Co-applicants: Louise Brunkwall (LU) 

  * Interneuronal populations in locomotor networks  PI: Ole Kiehn (KI) 

Application round: Feb 2014 

Subject: Cell and Developmental Biology

  * Intragenomic conflict  PI: Hanna Johannesson (UU) 

Application round: Jan 2017 

Subject: Evolutionary Biology/Genetics

Publications:

[ Hosseini _et al._ **Comparative analysis of genome-wide DNA methylation in Neurospora.** _Epigenetics_ 15:972-987 (2020) ](<http://dx.doi.org/10.1080/15592294.2020.1741758>)

  * Is DNA methylation heritable in clonal water fleas?  PI: Tobias Uller (LU) 

Application round: Jan 2018 

Subject: Evolutionary Biology/Genetics

  
Co-applicants: Reinder Radersma (LU) 

  * Long-term evolution of germline-restricted chromosomes in songbirds  PI: Alexander Suh (UU) 

Application round: Jan 2020 

Subject: Evolutionary Biology/Genetics

  
Co-applicants: Francisco Ruiz-Ruano (UU) 

  * Longitudinal proteomic profiling in blood and airways of COVID-19 patients  PI: Anna Smed Sörensen (KI) 

Application round: Oct 2020 

Subject: Molecular Medicine & Biomarkers

  * Machine learning approach to spatially resolved sequencing in brain tissue  PI: Tomas Björklund (LU) 

Application round: Feb 2019 

Subject: New Technology Development

  * Massive and parallel single-cell sequencing of eukaryotic plankton  PI: Anders Andersson (KTH) 

Application round: Oct 2020 

Subject: New Technology Development

  
Co-applicants: Rachel Foster (SU) 

  * Microbes in the deep  PI: Mark Dopson (LNU) 

Application round: Jan 2017 

Subject: Biodiversity

Tools and scripts:

[ **Metagenomics pipeline** – a Snakemake workflow for processing and analysis of metagenomic sequence reads ](<https://bitbucket.org/scilifelab-lts/nbis-meta>)

  * Mitochondrial dysfunction in Parkinson's disease  PI: Nils-Göran Larsson (KI) 

Application round: May 2017 

Subject: Molecular Medicine & Biomarkers

  * Molecular pathogenesis of liver lipotoxicity  PI: Margit Mahlapuu (GU) 

Application round: May 2020 

Subject: Molecular Medicine & Biomarkers

  * Multiomics analysis of response to epigenetic treatments in multiple myeloma  PI: Helena Jernberg Wiklund (UU) 

Application round: Jan 2020 

Subject: Cancer Biology

  
Co-applicants: Jan Komorowski (UU) 

  * Mutation calling to enable sensitive proteogenomics for cancer immunotherapy  PI: Janne Lehtiö (KI) 

Application round: Oct 2018 

Subject: New Technology Development

  
Co-applicants: Rui Branca (KI) 

  * Non-coding regulatory RNAs in the transition from uni- to multicellularity  PI: Fredrik Söderbom (UU) 

Application round: May 2020 

Subject: Cell and Developmental Biology

  * Novel regulatory patterns of biofilm formation in Candida parapsilosis  PI: Ute Römling (KI) 

Application round: May 2018 

Subject: Cell and Developmental Biology

  * Omics of sleep loss  PI: Christian Benedict (UU) 

Application round: Oct 2016 

Subject: Cell and Developmental Biology

  
Co-applicants: Jonathan Cedernaes (UU) 

Publications:

[ Cedernaes _et al._ **Acute sleep loss results in tissue-specific alterations in genome-wide DNA methylation state and metabolic fuel utilization in humans.** _Sci Adv_ 4:eaar8590 (2018) ](<http://dx.doi.org/10.1126/sciadv.aar8590>)

Tools and scripts:

[ **RNA-seq and DNA methylation analysis** – R scripts and output ](<https://github.com/orzechoj/cedernaes2018_analysis/tree/master>)

Data: [RNA-seq data](<https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6903/>) [DNA methylation data](<https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6908/>)

  * Oral microbiota and immune dynamics in early cancer protection  PI: Margaret Sällberg Chen (KI) 

Application round: Oct 2019 

Subject: Cancer Biology

Publications:

[ Morgell _et al._ **Metabolic characterization of plasma and cyst fluid from cystic precursors to pancreatic cancer patients reveal metabolic signatures of bacterial infection.** _J Proteome Res_ 20:2725-2738 (2021) ](<http://dx.doi.org/10.1021/acs.jproteome.1c00018>)

  * Organ-specific features of the lymphatic vasculature in the gastrointestinal tract  PI: Taija Mäkinen (UU) 

Application round: May 2020 

Subject: Cell and Developmental Biology

  * Pericyte fibrosis  PI: Christian Göritz (KI) 

Application round: Jun 2016 

Subject: Cell and Developmental Biology

Tools and scripts:

[ **Single-cell RNA-seq pipeline** – a Snakemake workflow for quality control and basic analysis of single-cell RNA-seq data (under development) ](<https://bitbucket.org/scilifelab-lts/lts-workflows-sm-scrnaseq/branch/develop>)

  * Pneumonial metagenomics, metatranscriptomics and metabolomics  PI: Birgitta Henriques-Normark (KI) 

Application round: Jun 2016 

Subject: Molecular Medicine & Biomarkers

Tools and scripts:

[ **Metagenomics pipeline** – a Snakemake workflow for processing and analysis of metagenomic sequence reads ](<https://bitbucket.org/scilifelab-lts/nbis-meta>)

  * Preimplantation stress: programming our future generations  PI: Sophie Petropoulos (KI) 

Application round: Jan 2017 

Subject: Cell and Developmental Biology

  * Profiling of drug treatments on sarcoma and coronavirus-infected cell lines  PI: Ola Spjuth (UU) 

Application round: May 2020 

Subject: Molecular Medicine & Biomarkers

  
Co-applicants: Sven Nelander (UU), Jordi Carreras-Puigvert (KI) 

  * Prophylactic probiotics to extremely low birth weight preterm infants  PI: Thomas Abrahamsson (LiU) 

Application round: Oct 2018 

Subject: Molecular Medicine & Biomarkers

  
Co-applicants: Maria Jenmalm (LiU) 

Tools and scripts:

[ **Metagenomics pipeline** – a Snakemake workflow for processing and analysis of metagenomic sequence reads ](<https://bitbucket.org/scilifelab-lts/nbis-meta>)

  * Proteogenomics of radiotherapy resistance in locally recurring breast cancer  PI: Emma Niméus (LU) 

Application round: Jan 2020 

Subject: Cancer Biology

  
Co-applicants: Lars Malmström (Univ. of Zurich) 

  * Quantifying the dynamics of the blood plasma proteome during acute HIV  PI: Joakim Esbjörnsson (LU) 

Application round: Jan 2021 

Subject: Molecular Medicine & Biomarkers

  
Co-applicants: Jamirah Nazziwa (LU) 

  * Redefining the molecular landscape of triple-negative breast cancer  PI: Johan Staaf (LU) 

Application round: Oct 2018 

Subject: Cancer Biology

  * Reproductive isolation and the evolution of key innovations  PI: Charlie Cornwallis (LU) 

Application round: Feb 2015 

Subject: Evolutionary Biology/Genetics

  * Respiratory genomics in flies and mammals  PI: Christos Samakovlis (SU) 

Application round: Nov 2014 

Subject: Cell and Developmental Biology

  * Seqitope - a novel tool for mapping antibody epitopes  PI: Mats Persson (KI) 

Application round: Oct 2020 

Subject: New Technology Development

  
Co-applicants: Anders Andersson (KTH) 

  * Sex-biased gene expression and its dependence on condition  PI: Urban Friberg (LiU) 

Application round: May 2018 

Subject: Evolutionary Biology/Genetics

  * Sex-limited experimental evolution of novel sex chromosomes  PI: Jessica Abbott (LU) 

Application round: May 2018 

Subject: Evolutionary Biology/Genetics

  * Single cell profiling of cell types and lineages in the mouse brain  PI: Jonas Frisén (KI) 

Application round: Jan 2018 

Subject: New Technology Development

  * Single-cell analysis of SARS-CoV-2 infection in a 3D primary cell lung  PI: Johan Henriksson (UmU) 

Application round: Jan 2021 

Subject: Molecular Medicine & Biomarkers

  
Co-applicants: Anna Överby Wernstedt (UmU), Annasara Lenman (UmU) 

  * Single-cell gene expression analysis of limb regeneration  PI: Andras Simon (KI) 

Application round: Jun 2015 

Subject: Cell and Developmental Biology

  * Somatic mutations and its implications for age-related disease  PI: Maria Eriksson (KI) 

Application round: Oct 2020 

Subject: Human Genetics

  * SuperGenE - supergene evolution in a classic plant system  PI: Tanja Slotte (SU) 

Application round: Feb 2019 

Subject: Evolutionary Biology/Genetics

  * Symbiont-induced host speciation in insects  PI: Lisa Klasson (UU) 

Application round: Jan 2017 

Subject: Evolutionary Biology/Genetics

  * Systems biology dynamics of liver regeneration in humans  PI: Niklas Björkström (KI) 

Application round: Oct 2019 

Subject: Cell and Developmental Biology

  
Co-applicants: Martin Cornillet (KI) 

  * Targeting topoisomerase regulation as a mean to control transcription in cancer cells  PI: Laura Baranello (KI) 

Application round: Oct 2019 

Subject: Cancer Biology

  * The basis of inherited migratory directions in the Australian bogong moth  PI: Eric Warrant (LU) 

Application round: May 2018 

Subject: Evolutionary Biology/Genetics

  * The epigenetic landscape in MLL-rearranged leukemia  PI: Anna Andersson (LU) 

Application round: Feb 2016 

Subject: Cancer Biology

  * The genomics and transcriptomics behind Nature&aposs high performance material  PI: Anna Rising (KI) 

Application round: Oct 2018 

Subject: Cell and Developmental Biology

  * The molecular basis of recombination rate variation in butterflies  PI: Niclas Backström (UU) 

Application round: Oct 2019 

Subject: Evolutionary Biology/Genetics

  * The role for gene repression in maintaining differentiated identity  PI: Johan Holmberg (KI) 

Application round: May 2019 

Subject: Cell and Developmental Biology

  * The transcriptional landscape of mucosal healing  PI: Eduardo Villablanca (KI) 

Application round: May 2020 

Subject: Molecular Medicine & Biomarkers

  * Towards tailored treatments in multiple sclerosis  PI: Joachim Burman (UU) 

Application round: Oct 2019 

Subject: Molecular Medicine & Biomarkers

  
Co-applicants: Kim Kultima (UU) 

  * Tracing human history in southern Africa using ancient proteomics  PI: Carina Schlebusch (UU) 

Application round: Jan 2021 

Subject: Evolutionary Biology/Genetics

  
Co-applicants: Imke Lankheet (UU) 

  * Transcriptional and immune receptor profiling of influenza-specific B cells  PI: Davide Angeletti (GU) 

Application round: Oct 2019 

Subject: Cell and Developmental Biology

Publications:

[ Matthew _et al._ **Single-cell BCR and transcriptome analysis after influenza infection reveals spatiotemporal dynamics of antigen-specific B cells.** _Cell Rep_ 35:109286 (2021) ](<http://dx.doi.org/10.1016/j.celrep.2021.109286>)

Tools and scripts:

[ **Data analysis scripts** – Code and supporting files for reproducing the analyses reported in the publication ](<https://github.com/angelettilab/scMouseBcellFlu>)

Data: [VDJ data](<https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9478/>) [Gene expression data](<https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9491/>)

  * Transcriptional profiling of mononuclear phagocytes in sarcoidosis patients  PI: Anna Smed Sörensen (KI) 

Application round: May 2018 

Subject: Molecular Medicine & Biomarkers

Publications:

[ Lepzien _et al._ **Monocytes in sarcoidosis are potent TNF producers and predict disease outcome.** _Eur Respir J_ 2003468 (2021) ](<http://dx.doi.org/10.1183/13993003.03468-2020>)

  * Transcriptome analysis of memory T cells during influenza virus challenge  PI: Nils Lycke (GU) 

Application round: Jan 2018 

Subject: Cell and Developmental Biology

  * Transcriptome analysis of the response to mechanical stimulation in cereals  PI: Olivier van Aken (LU) 

Application round: Jan 2020 

Subject: Biodiversity

  
Co-applicants: Nick Sirijovski (LU), Essam Darwish (LU) 

  * Understanding epigenetics of type 2 diabetes by WGBS  PI: Charlotte Ling (LU) 

Application round: Jan 2018 

Subject: Molecular Medicine & Biomarkers

  
Co-applicants: Alexander Perfilyev (LU) 

  * Understanding the tissue-material interface during wound healing  PI: Molly Stevens (KI) 

Application round: Jan 2018 

Subject: Molecular Medicine & Biomarkers

  
Co-applicants: Alessondra Speidel (KI), Annina Steinbach (KI) 

  * WGS and RNA-seq in chronic lymphocytic leukemia subsets  PI: Richard Rosenquist (UU) 

Application round: Jun 2016 

Subject: Cancer Biology

  * Whole genome sequencing to identify genetic causes of adverse drug reactions  PI: Mia Wadelius (UU) 

Application round: Sep 2017 

Subject: Human Genetics

  * Whole-genome sequencing of 1000 colorectal cancers  PI: Tobias Sjöblom (UU) 

Application round: Oct 2018 

Subject: Cancer Biology

  
Co-applicants: Mathias Uhlén (KTH) 

  * Why such high genetic diversity in songbird immunity?  PI: Helena Westerdahl (LU) 

Application round: Jun 2016 

Subject: Evolutionary Biology/Genetics

  * scRNA-seq analysis of B cells in patients with Sjögren&aposs Syndrome  PI: Ann-Christine Syvänen (UU) 

Application round: Jan 2021 

Subject: Molecular Medicine & Biomarkers

  
Co-applicants: Gunnel Nordmark (UU), Gustav Arvidsson (UU), Amanda Raine (UU) 



    
    
      Organization,No of projects
      GU, 4
      KI, 36
      KTH, 4
      LNU, 1
      LU, 17
      LiU, 2
      NRM, 1
      SU, 7
      UU, 25
      UmU, 1
      

Click on a project for further details and links to publications, analysis tools and data.

_unfold_more_ Show all details _unfold_less_ Hide all details

  * Acoelomorph genomes  PI: Ulf Jondelius (NRM) 

Application round: Jun 2015 

Subject: Evolutionary Biology/Genetics

  * Analysis of the nuclear and cytosolic transcriptomes in human brain  PI: Lars Feuk (UU) 

Application round: Feb 2015 

Subject: New Technology Development

Publications:

[ Zaghlool _et al._ **Expression profiling and in situ screening of circular RNAs in human tissues.** _Sci Rep_ 8:16953 (2018) ](<http://dx.doi.org/10.1038/s41598-018-35001-6>) [ Zaghlool _et al._ **Characterization of the nuclear and cytosolic transcriptomes in human brain tissue reveals new insights into the subcellular distribution of RNA transcripts.** _Sci Rep_ 11:4076 (2021) ](<http://dx.doi.org/10.1038/s41598-021-83541-1>)

  * Ancient DNA metagenome  PI: Barbara Wohlfarth (SU) 

Application round: Jun 2016 

Subject: Biodiversity

Publications:

[ Ahmed _et al._ **Archaeal community changes in lateglacial lake sediments: Evidence from ancient DNA.** _Quat Sci Rev_ 181:19-29 (2018) ](<http://dx.doi.org/10.1016/j.quascirev.2017.11.037>) [ Parducci _et al._ **Shotgun environmental DNA, pollen, and macrofossil analysis of lateglacial lake sediments from southern Sweden.** _Front Ecol Evol_ 7:189 (2019) ](<http://dx.doi.org/10.3389/fevo.2019.00189>)

Data: [DNA sequence reads](<https://www.ebi.ac.uk/ena/data/view/PRJNA378719>)

  * Antibody repertoires  PI: Gunilla Karlsson Hedestam (KI) 

Application round: May 2014 

Subject: New Technology Development

Publications:

[ Corcoran _et al._ **Production of individualized V gene databases reveals high levels of immunoglobulin genetic diversity.** _Nat Commun_ 7:13642 (2016) ](<http://dx.doi.org/10.1038/ncomms13642>)

Tools and scripts:

[ **IgDiscover** – Software for antibody repertoire analysis and V gene discovery from high-throughput sequencing reads ](<http://docs.igdiscover.se/>)

  * Bacterial regulatory RNAs  PI: Emmanuelle Charpentier (UmU) 

Application round: Oct 2013 

Subject: Evolutionary Biology/Genetics

Publications:

[ Le Rhun _et al._ **RNA sequencing uncovers antisense RNAs and novel small RNAs in _Streptococcus pyogenes_.** _RNA Biol_ 13:177-195 (2015) ](<http://dx.doi.org/10.1080/15476286.2015.1110674>) [ Le Rhun _et al._ **Identification of endoribonuclease specific cleavage positions reveals novel targets of RNase III in _Streptococcus pyogenes_.** _Nucleic Acids Res_ gkw1316 (2017) ](<http://dx.doi.org/10.1093/nar/gkw1316>)

Data: [RNA-seq data set 1](<https://www.ncbi.nlm.nih.gov/bioproject/PRJNA341673>) [RNA-seq data set 2](<https://www.ncbi.nlm.nih.gov/bioproject/PRJNA341675>)

  * Beetle genome assembly and annotation  PI: Göran Arnqvist (UU) 

Application round: May 2013 

Subject: Evolutionary Biology/Genetics

Publications:

[ Sayadi _et al._ **The evolution of dark matter in the mitogenome of seed beetles.** _Genome Biol Evol_ 9:2697-2706 (2017) ](<http://dx.doi.org/10.1093/gbe/evx205>) [ Sayadi _et al._ **The genomic footprint of sexual conflict.** _Nat Ecol Evol_ 3:1725-1730 (2019) ](<http://dx.doi.org/10.1038/s41559-019-1041-9>)

  * Biodiversity in air  PI: Per Stenberg (UmU) 

Application round: Jun 2016 

Subject: Biodiversity

Tools and scripts:

[ **Metagenomics pipeline** – a snakemake workflow for processing and analysis of metagenomic sequence reads ](<https://bitbucket.org/scilifelab-lts/nbis-meta>)

  * Body weight sensor  PI: Claes Ohlsson (GU) 

Application round: Oct 2016 

Subject: Cell and Developmental Biology

  * Cancer proteogenomics  PI: Janne Lehtiö (KI) 

Application round: Nov 2014 

Subject: Cancer Biology

Publications:

[ Zhu _et al._ **Discovery of coding regions in the human genome by integrated proteogenomics analysis workflow.** _Nat Commun_ 9:903 (2018) ](<http://dx.doi.org/10.1038/s41467-018-03311-y>) [ Johansson _et al._ **Breast cancer quantitative proteome and proteogenomic landscape.** _Nat Commun_ 10:1600 (2019) ](<http://dx.doi.org/10.1038/s41467-019-09018-y>)

Data: [Proteomics data for A431 cells and tissues](<http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD006291>) [Proteomics data for breast tumors](<http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD008841>) [Proteomics data for tumor and matched normal tissue](<http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD011385>) [RNA-seq data for A431 cells](<https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5285>) [RNA-seq data for tissues](<https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2836>)

  * Colon cancer genes  PI: Erik Johansson (UmU) 

Application round: Feb 2014 

Subject: Cancer Biology

Publications:

[ Rentoft _et al._ **A geographically matched control population efficiently limits the number of candidate disease-causing variants in an unbiased whole-genome analysis.** _PLoS One_ 14:e0213350 (2019) ](<http://dx.doi.org/10.1371/journal.pone.0213350>)

  * DACP rat genome sequencing  PI: Ulrika Norin (KI) 

Application round: Jan 2013 

Subject: Molecular Medicine & Biomarkers

Publications:

[ Norin _et al._ **Endophilin A2 deficiency protects rodents from autoimmune arthritis by modulating T cell activation.** _Nat Commun_ 12:610 (2021) ](<http://dx.doi.org/10.1038/s41467-020-20586-2>)

  * Development and diversity of midbrain dopamine neurons  PI: Thomas Perlmann (KI) 

Application round: Feb 2014 

Subject: Cell and Developmental Biology

Publications:

[ Kee _et al._ **Single-cell analysis reveals a close relationship between differentiating dopamine and subthalamic nucleus neuronal lineages.** _Cell Stem Cell_ 20:29-40 (2017) ](<http://dx.doi.org/10.1016/j.stem.2016.10.003>) [ Tiklová _et al._ **Single-cell RNA sequencing reveals midbrain dopamine neuron diversity emerging during mouse brain development.** _Nat Commun_ 10:581 (2019) ](<http://dx.doi.org/10.1038/s41467-019-08453-1>)

Data: [RNA-seq data set 1](<https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87069>) [RNA-seq data set 2](<https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116138>)

  * Diabetogenic perturbations of the fetal methylome  PI: Paul Franks (LU) 

Application round: Jan 2017 

Subject: New Technology Development

  * Diseases in human history  PI: Kristian Kristiansen (GU) 

Application round: Oct 2015 

Subject: Biodiversity

Tools and scripts:

[ **Metagenomics pipeline** – a snakemake workflow for processing and analysis of metagenomic sequence reads ](<https://bitbucket.org/scilifelab-lts/nbis-meta>)

  * Endurance training and gene expression in human muscle  PI: Carl-Johan Sundberg (KI) 

Application round: Jan 2013 

Subject: Cell and Developmental Biology

Publications:

[ Lindholm _et al._ **The human skeletal muscle transcriptome: sex differences, alternative splicing, and tissue homogeneity assessed with RNA sequencing.** _FASEB J_ 28:4571-4581 (2014) ](<http://dx.doi.org/10.1096/fj.14-255000>) [ Lindskog _et al._ **The human cardiac and skeletal muscle proteomes defined by transcriptomics and antibody-based profiling.** _BMC Genomics_ 16:475 (2015) ](<http://dx.doi.org/10.1186/s12864-015-1686-y>) [ Lindholm _et al._ **The impact of endurance training on human skeletal muscle memory, global isoform expression and novel transcripts.** _PLoS Genet_ 12:e1006294 (2016) ](<http://dx.doi.org/10.1371/journal.pgen.1006294>)

Data: [RNA-seq data set 1](<https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE58387>) [RNA-seq data set 2](<https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE58608>) [RNA-seq data set 3](<https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60590>) [RNA-seq data set 4](<https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60833>)

  * Epigenetic states of oligodendrocyte lineage cells  PI: Gonçalo Castelo-Branco (KI) 

Application round: Feb 2016 

Subject: Cell and Developmental Biology

Publications:

[ Marques _et al._ **Transcriptional convergence of oligodendrocyte lineage progenitors during development.** _Dev Cell_ 46:504-517 (2018) ](<http://dx.doi.org/10.1016/j.devcel.2018.07.005>) [ Falcão _et al._ **Disease-specific oligodendrocyte lineage cells arise in multiple sclerosis.** _Nat Med_ 24:1837-1844 (2018) ](<http://dx.doi.org/10.1038/s41591-018-0236-y>)

Tools and scripts:

[ **OPCs throughout mouse development** – Web resource for exploring gene expression data from Marques et al. ](<https://castelobranco.shinyapps.io/OPCsinglecell2017/>) [ **Oligodendrocyte lineage in a mouse model of multiple sclerosis** – Web resource for exploring gene expression data from Falcão et al. ](<https://castelobranco.shinyapps.io/EAE2017/>)

Data: [Bulk RNA-seq data](<https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95093>) [Single-cell RNA-seq data 1](<https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95194>) [Single-cell RNA-seq data 2](<https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113973>)

  * Evolution of gene regulation in plants  PI: Tanja Slotte (UU) 

Application round: Jan 2013 

Subject: Evolutionary Biology/Genetics

Publications:

[ Steige _et al._ **_Cis_ -regulatory changes associated with a recent mating system shift and floral adaptation in _Capsella_.** _Mol Biol Evol_ 32:2501-2514 (2015) ](<http://dx.doi.org/10.1093/molbev/msv169>) [ Steige _et al._ **The role of transposable elements for gene expression in _Capsella_ hybrids and allopolyploids.** _bioRxiv_ 44016 (2016) ](<http://dx.doi.org/10.1101/044016>) [ Steige _et al._ **Genomic analysis reveals major determinants of _cis_ -regulatory variation in _Capsella grandiflora_.** _Proc Natl Acad Sci USA_ 114:1087-1092 (2017) ](<http://dx.doi.org/10.1073/pnas.1612561114>)

  * Examination of embryonic and adult stem cell characteristics  PI: Jonas Muhr (KI) 

Application round: May 2014 

Subject: Cell and Developmental Biology

Publications:

[ Hagey _et al._ **Distinct transcription factor complexes act on a permissive chromatin landscape to establish regionalized gene expression in CNS stem cells.** _Genome Res_ 26:908-917 (2016) ](<http://dx.doi.org/10.1101/gr.203513.115>) [ Klum _et al._ **Sequentially acting SOX proteins orchestrate astrocyte‐ and oligodendrocyte‐specific gene expression.** _EMBO Rep_ e46635 (2018) ](<http://dx.doi.org/10.15252/embr.201846635>)

  * Exercise-regulated mechanisms in mouse organs  PI: Pontus Boström (KI) 

Application round: Feb 2014 

Subject: Cell and Developmental Biology

  * Exploring sex-biased gene expression in willows  PI: Sofia Berlin Kolm (SLU) 

Application round: Oct 2015 

Subject: Evolutionary Biology/Genetics

Publications:

[ Almeida _et al._ **Genome assembly of the basket willow, Salix viminalis, reveals earliest stages of sex chromosome expansion.** _BMC Biol_ 18:78 (2020) ](<http://dx.doi.org/10.1186/s12915-020-00808-1>)

  * Finding causal variants within schizophrenia risk loci  PI: Patrick Sullivan (KI) 

Application round: Oct 2016 

Subject: Human Genetics

Publications:

[ Halvorsen _et al._ **Increased burden of ultra-rare structural variants localizing to boundaries of topologically associated domains in schizophrenia.** _Nat Commun_ 11:1842 (2020) ](<http://dx.doi.org/10.1038/s41467-020-15707-w>)

  * Finding new treatments for type 2 diabetes  PI: Nils Wierup (LU) 

Application round: Feb 2015 

Subject: Molecular Medicine & Biomarkers

  * Flycatcher genomics  PI: Hans Ellegren (UU) 

Application round: Jan 2013 

Subject: Evolutionary Biology/Genetics

Publications:

[ Burri _et al._ **Linked selection and recombination rate variation drive the evolution of the genomic landscape of differentiation across the speciation continuum of _Ficedula_ flycatchers.** _Genome Res_ 25:1656-1665 (2015) ](<http://dx.doi.org/10.1101/gr.196485.115>)

Data: [Reference genome assembly](<http://www.ebi.ac.uk/ena/data/view/PRJNA75089>) [Genotypes for linkage analysis](<http://dx.doi.org/10.5061/dryad.v0v83>) [Resequencing data](<http://www.ebi.ac.uk/ena/data/view/PRJEB7359>)

  * Fungal genome annotation through proteogenomics  PI: Annika Scheynius (KI) 

Application round: May 2013 

Subject: New Technology Development

Publications:

[ Zhu _et al._ **Proteogenomics produces comprehensive and highly accurate protein-coding gene annotation in a complete genome assembly of _Malassezia sympodialis_.** _Nucleic Acids Res_ 45:2629-2643 (2017) ](<http://dx.doi.org/10.1093/nar/gkx006>)

Data: [DNA sequence reads, genome assemblies and gene annotation](<https://www.ebi.ac.uk/ena/data/view/PRJEB13283>) [RNA-seq data](<https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-4589>) [Proteomics data](<https://www.ebi.ac.uk/pride/archive/projects/PXD003773>)

  * Gene regulatory networks in β-cells  PI: Olov Andersson (KI) 

Application round: Sep 2017 

Subject: Cell and Developmental Biology

Publications:

[ Charbord _et al._ **In vivo screen identifies a SIK inhibitor that induces β cell proliferation through a transient UPR.** _Nat Metab_ 3:682-700 (2021) ](<http://dx.doi.org/10.1038/s42255-021-00391-x>)

Tools and scripts:

[ **Data exploration** – Web resource for exploring gene expression data from Charbord et al. ](<http://rshiny.nbis.se/shiny-server-apps/o_andersson/>)

Data: [RNA-seq data](<https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE163972>)

  * Genetic associations of serum lipidome in a prospective cohort  PI: Olle Melander (LU) 

Application round: May 2014 

Subject: Molecular Medicine & Biomarkers

Publications:

[ Kjellqvist _et al._ **Identification of shared and unique serum lipid profiles in diabetes mellitus and myocardial infarction.** _J Am Heart Assoc_ 5:e004503 (2016) ](<http://dx.doi.org/10.1161/jaha.116.004503>) [ Fernandez _et al._ **Plasma lipidome and prediction of type 2 diabetes in the population-based Malmö diet and cancer cohort.** _Dia Care_ 43:366-373 (2020) ](<http://dx.doi.org/10.2337/dc19-1199>) [ Ottosson _et al._ **A plasma lipid signature predicts incident coronary artery disease.** _Int J Cardiol_ 331:249-254 (2021) ](<http://dx.doi.org/10.1016/j.ijcard.2021.01.059>)

  * Genetic risk factors for ANCA-associated vasculitides  PI: Kerstin Lindblad-Toh (UU) 

Application round: Feb 2015 

Subject: Human Genetics

  * Genetic variation of European aspen  PI: Pär Ingvarsson (UmU) 

Application round: Oct 2013 

Subject: Evolutionary Biology/Genetics

Publications:

[ Wang _et al._ **A major locus controls local adaptation and adaptive life history variation in a perennial plant.** _Genome Biol_ 19:72 (2018) ](<http://dx.doi.org/10.1186/s13059-018-1444-y>) [ Lin _et al._ **Functional and evolutionary genomic inferences in Populus through genome and population sequencing of American and European aspen.** _Proc Natl Acad Sci USA_ 115:E10970-E10978 (2018) ](<http://dx.doi.org/10.1073/pnas.1801437115>)

  * Genetics of heart stress recovery  PI: Jan Borén (GU) 

Application round: Oct 2013 

Subject: Human Genetics

  * Genome-wide epigenetic variation in type 2 diabetes  PI: Charlotte Ling (LU) 

Application round: Nov 2014 

Subject: Human Genetics

Publications:

[ Bysani _et al._ **ATAC-seq reveals alterations in open chromatin in pancreatic islets from subjects with type 2 diabetes.** _Sci Rep_ 9:7785 (2019) ](<http://dx.doi.org/10.1038/s41598-019-44076-8>)

Data: [Gene expression data](<https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE50398>) [ATAC-seq data](<https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129383>)

  * Hemostatic regulation in human liver  PI: Christina Jern (GU) 

Application round: May 2014 

Subject: Human Genetics

Publications:

[ Olsson Lindvall _et al._ **Hemostatic genes exhibit a high degree of allele-specific regulation in liver.** _Thromb Haemost_ 119:1072-1083 (2019) ](<http://dx.doi.org/10.1055/s-0039-1687879>) [ Olsson Lindvall _et al._ **A comprehensive sequencing-based analysis of allelic methylation patterns in hemostatic genes in human liver.** _Thromb Haemost_ 120:229-242 (2020) ](<http://dx.doi.org/10.1055/s-0039-3401824>) [ Olsson Lindvall _et al._ **Comparison of DNA Methylation Profiles of Hemostatic Genes between Liver Tissue and Peripheral Blood within Individuals.** _Thromb Haemost_ 121:573-583 (2021) ](<http://dx.doi.org/10.1055/s-0040-1720980>)

  * Human structural genetic variation  PI: Magnus Nordenskjöld (KI) 

Application round: Nov 2014 

Subject: Human Genetics

Publications:

[ Eisfeldt _et al._ **TIDDIT, an efficient and comprehensive structural variant caller for massive parallel sequencing data.** _F1000Res_ 6:664 (2017) ](<http://dx.doi.org/10.12688/f1000research.11168.2>) [ Lindstrand _et al._ **From cytogenetics to cytogenomics: whole-genome sequencing as a first-line test comprehensively captures the diverse spectrum of disease-causing genetic variation underlying intellectual disability.** _Genome Med_ 11:68 (2019) ](<http://dx.doi.org/10.1186/s13073-019-0675-1>)

Tools and scripts:

[ **vcf2cytosure** – Convert structural variants from VCF format to the format used in CytoSure Interpret ](<https://github.com/NBISweden/vcf2cytosure>)

  * Impacts of nitrogen fertilisation on forest belowground fungal communities  PI: Nathaniel Street (UmU) 

Application round: Jan 2018 

Subject: Biodiversity

  
Co-applicants: Vaughan Hurry (SLU) 

Publications:

[ Schneider _et al._ **Comparative Fungal Community Analyses Using Metatranscriptomics and Internal Transcribed Spacer Amplicon Sequencing from Norway Spruce.** _mSystems_ 6:e00884-20 (2021) ](<http://dx.doi.org/10.1128/msystems.00884-20>)

  * Improving transcriptome reconstruction  PI: Olof Emanuelsson (KTH) 

Application round: May 2013 

Subject: Evolutionary Biology/Genetics

Publications:

[ Reimegård _et al._ **Genome-wide identification of physically clustered genes suggests chromatin-level co-regulation in male reproductive development in _Arabidopsis thaliana_.** _Nucleic Acids Res_ 45:3253-3265 (2017) ](<http://dx.doi.org/10.1093/nar/gkx087>)

  * Integrated multiomics analysis of pediatric acute myeloid leukemia  PI: Linda Holmfeldt (UU) 

Application round: Jan 2018 

Subject: Cancer Biology

Publications:

[ Stratmann _et al._ **Genomic characterization of relapsed acute myeloid leukemia reveals novel putative therapeutic targets.** _Blood Adv_ 5:900-912 (2021) ](<http://dx.doi.org/10.1182/bloodadvances.2020003709>)

  * Iron metabolism and inflammation  PI: Anna-Lena Spetz (SU) 

Application round: Oct 2015 

Subject: Human Genetics

  * Leptidea butterfly genomics  PI: Niclas Backström (UU) 

Application round: Feb 2015 

Subject: Evolutionary Biology/Genetics

Publications:

[ Talla _et al._ **Lack of gene flow: Narrow and dispersed differentiation islands in a triplet of Leptidea butterfly species.** _Mol Ecol_ 28:3756-3770 (2019) ](<http://dx.doi.org/10.1111/mec.15188>)

Data: [Leptidea DNA sequence reads, genome assembly and variant calls](<https://www.ebi.ac.uk/ena/data/view/PRJEB21838>)

  * Marine genomics of barnacle  PI: Anders Blomberg (GU) 

Application round: Jan 2013 

Subject: Biodiversity

Publications:

[ Alm Rosenblad _et al._ **Genomic characterization of the barnacle _Balanus improvisus_ reveals extreme nucleotide diversity in coding regions.** _Mar Biotechnol_ advance online publication (2021) ](<http://dx.doi.org/10.1007/s10126-021-10033-8>)

Data: [Pilot genome assembly](<https://doi.org/10.17044/scilifelab.14339153>)

  * Melanoma genetics  PI: Veronica Höiom (KI) 

Application round: Jan 2013 

Subject: Human Genetics

Publications:

[ Tuominen _et al._ **The role of germline alterations in the DNA damage response genes _BRIP1_ and _BRCA2_ in melanoma susceptibility.** _Genes Chromosomes Cancer_ 55:601-611 (2016) ](<http://dx.doi.org/10.1002/gcc.22363>)

  * Methods for multiplex protein analysis  PI: Ulf Landegren (UU) 

Application round: May 2017 

Subject: New Technology Development

  
Co-applicants: Caroline Gallant (UU) 

Publications:

[ Reimegård _et al._ **A combined approach for single-cell mRNA and intracellular protein expression analysis.** _Commun Biol_ 4:624 (2021) ](<http://dx.doi.org/10.1038/s42003-021-02142-w>)

Tools and scripts:

[ **Scripts and data files** – to generate figures and tables from the article ](<https://github.com/b97jre/SPARC>)

  * Methylation QTL analysis in the chicken  PI: Dominic Wright (LiU) 

Application round: Jun 2015 

Subject: Evolutionary Biology/Genetics

Publications:

[ Höglund _et al._ **The methylation landscape and its role in domestication and gene regulation in the chicken.** _Nat Ecol Evol_ 4:1713-1724 (2020) ](<http://dx.doi.org/10.1038/s41559-020-01310-1>)

  * MicroRNA in muscle dystrophy  PI: Madeleine Durbeej-Hjalt (LU) 

Application round: Nov 2014 

Subject: Molecular Medicine & Biomarkers

  * MicroRNA networks in Parkinson's disease  PI: Johan Jakobsson (LU) 

Application round: Oct 2015 

Subject: Cell and Developmental Biology

  * Multi-level omics profiling of the yeast cell cycle  PI: Jens Nielsen (CTH) 

Application round: Jun 2016 

Subject: Cell and Developmental Biology

Publications:

[ Campbell _et al._ **Building blocks are synthesized on demand during the yeast cell cycle.** _Proc Natl Acad Sci USA_ 117:7575-7583 (2020) ](<http://dx.doi.org/10.1073/pnas.1919535117>)

  * Multi-omics analysis of human neuronal cells in health and disease  PI: Niklas Dahl (UU) 

Application round: Feb 2016 

Subject: Cell and Developmental Biology

Publications:

[ Sobol _et al._ **Transcriptome and proteome profiling of neural induced pluripotent stem cells from individuals with Down syndrome disclose dynamic dysregulations of key pathways and cellular functions.** _Mol Neurobiol_ 56:7113-7127 (2019) ](<http://dx.doi.org/10.1007/s12035-019-1585-3>) [ Schuster _et al._ **Transcriptomes of Dravet syndrome iPSC derived GABAergic cells reveal dysregulated pathways for chromatin remodeling and neurodevelopment.** _Neurobiol Dis_ 132:104583 (2019) ](<http://dx.doi.org/10.1016/j.nbd.2019.104583>) [ Laan _et al._ **DNA methylation changes in Down syndrome derived neural iPSCs uncover co-dysregulation of ZNF and HOX3 families of transcription factors.** _Clin Epigenetics_ 12:9 (2020) ](<http://dx.doi.org/10.1186/s13148-019-0803-1>)

  * Mutations in childhood acute lymphoblastic leukemia  PI: Ann-Christine Syvänen (UU) 

Application round: May 2013 

Subject: Cancer Biology

Publications:

[ Lindqvist _et al._ **The mutational landscape in pediatric acute lymphoblastic leukemia deciphered by whole genome sequencing.** _Hum Mutat_ 36:118-128 (2015) ](<http://dx.doi.org/10.1002/humu.22719>) [ Lindqvist _et al._ **Deep targeted sequencing in pediatric acute lymphoblastic leukemia unveils distinct mutational patterns between genetic subtypes and novel relapse-associated genes.** _Oncotarget_ 7:64071-64088 (2016) ](<http://dx.doi.org/10.18632/oncotarget.11773>)

  * Neo-antigens in melanoma  PI: Göran Jönsson (LU) 

Application round: Oct 2015 

Subject: Cancer Biology

Publications:

[ Lauss _et al._ **Mutational and putative neoantigen load predict clinical benefit of adoptive T cell therapy in melanoma.** _Nat Commun_ 8:1738 (2017) ](<http://dx.doi.org/10.1038/s41467-017-01460-0>)

  * Neuronal identity  PI: Johan Holmberg (KI) 

Application round: Jun 2015 

Subject: Cell and Developmental Biology

Publications:

[ Södersten _et al._ **A comprehensive map coupling histone modifications with gene regulation in adult dopaminergic and serotonergic neurons.** _Nat Commun_ 9:1226 (2018) ](<http://dx.doi.org/10.1038/s41467-018-03538-9>)

Data: [RNA-seq, ChIP-seq and TRAP-seq data](<https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107656>)

  * New mutations in dilated cardiomyopathy  PI: Martin Bergö (GU) 

Application round: May 2013 

Subject: Human Genetics

Publications:

[ Dalin _et al._ **Massive parallel sequencing questions the pathogenic role of missense variants in dilated cardiomyopathy.** _Int J Cardiol_ 228:742-748 (2017) ](<http://dx.doi.org/10.1016/j.ijcard.2016.11.066>)

  * Next-gen plant phylogenetics  PI: Alexandre Antonelli (GU) 

Application round: Feb 2014 

Subject: Evolutionary Biology/Genetics

Publications:

[ Andermann _et al._ **SECAPR - a bioinformatics pipeline for the rapid and user-friendly processing of targeted enriched Illumina sequences, from raw reads to alignments.** _PeerJ_ 6:e5175 (2018) ](<http://dx.doi.org/10.7717/peerj.5175>)

Tools and scripts:

[ **SECAPR** – pipeline for processing of sequence capture data ](<https://github.com/AntonelliLab/seqcap_processor>)

  * Novel subpopulations in the brain reward-aversion system  PI: Åsa Mackenzie (UU) 

Application round: Oct 2015 

Subject: Cell and Developmental Biology

Publications:

[ Wallén-Mackenzie _et al._ **Spatio-molecular domains identified in the mouse subthalamic nucleus and neighboring glutamatergic and GABAergic brain structures.** _Commun Biol_ 3:338 (2020) ](<http://dx.doi.org/10.1038/s42003-020-1028-8>)

  * Omics interplay in cardiometabolic disease  PI: Marju Orho-Melander (LU) 

Application round: Jan 2017 

Subject: Molecular Medicine & Biomarkers

  * Origin and heterogeneity of the most primitive nervous system  PI: Igor Adameyko (KI) 

Application round: Jun 2016 

Subject: Evolutionary Biology/Genetics

Publications:

[ Klimovich _et al._ **Prototypical pacemaker neurons interact with the resident microbiota.** _Proc Natl Acad Sci USA_ 117:17854-17863 (2020) ](<http://dx.doi.org/10.1073/pnas.1920469117>)

Tools and scripts:

[ **Single-cell RNA-seq pipeline** – a Snakemake workflow for quality control and basic analysis of single-cell RNA-seq data (under development) ](<https://bitbucket.org/scilifelab-lts/lts-workflows-sm-scrnaseq/branch/develop>)

  * Personalized profiling of adaptive immune genes  PI: Gunilla Karlsson Hedestam (KI) 

Application round: May 2017 

Subject: New Technology Development

Publications:

[ Vázquez Bernat _et al._ **High-quality library preparation for NGS-based immunoglobulin germline gene inference and repertoire expression analysis.** _Front Immunol_ 10:660 (2019) ](<http://dx.doi.org/10.3389/fimmu.2019.00660>) [ Phad _et al._ **Extensive dissemination and intraclonal maturation of HIV Env vaccine-induced B cell responses.** _J Exp Med_ 217:e20191155 (2020) ](<http://dx.doi.org/10.1084/jem.20191155>)

Tools and scripts:

[ **IgDiscover** – Software for antibody repertoire analysis and V gene discovery from high-throughput sequencing reads ](<http://docs.igdiscover.se/>)

  * Plant spatial transcriptomics  PI: Nathaniel Street (UmU) 

Application round: Feb 2016 

Subject: New Technology Development

Publications:

[ Giacomello _et al._ **Spatially resolved transcriptome profiling in model plant species.** _Nat Plants_ 3:17061 (2017) ](<http://dx.doi.org/10.1038/nplants.2017.61>)

  * SCAPIS human wellness profiling  PI: Mathias Uhlén (KTH) 

Application round: Feb 2016 

Subject: Cell and Developmental Biology

Publications:

[ Tebani _et al._ **Integration of molecular profiles in a longitudinal wellness profiling cohort.** _Nat Commun_ 11:4487 (2020) ](<http://dx.doi.org/10.1038/s41467-020-18148-7>)

  * Sequencing the missing link in land plant evolution  PI: Hans Ronne (SLU) 

Application round: Oct 2013 

Subject: Evolutionary Biology/Genetics

  * Single nuclei sequencing of arbuscular mycorrhizal fungi  PI: Anna Rosling (UU) 

Application round: May 2017 

Subject: Biodiversity

Publications:

[ Montoliu-Nerin _et al._ **Building de novo reference genome assemblies of complex eukaryotic microorganisms from single nuclei.** _Sci Rep_ 10:1303 (2020) ](<http://dx.doi.org/10.1038/s41598-020-58025-3>)

  * Somatic genetic variation in the aging brain and Alzheimer's disease  PI: Maria Eriksson (KI) 

Application round: Oct 2015 

Subject: Human Genetics

Publications:

[ Helgadottir _et al._ **Somatic mutation that affects transcription factor binding upstream of CD55 in the temporal cortex of a late-onset Alzheimer disease patient.** _Hum Mol Genet_ 28:2675–2685 (2019) ](<http://dx.doi.org/10.1093/hmg/ddz085>)

  * Somatic mutations in tissue stem cells during human aging  PI: Maria Eriksson (KI) 

Application round: Jun 2015 

Subject: Human Genetics

Publications:

[ Franco _et al._ **Somatic mutagenesis in satellite cells associates with human skeletal muscle aging.** _Nat Commun_ 9:800 (2018) ](<http://dx.doi.org/10.1038/s41467-018-03244-6>)

  * Spatial transcripts  PI: Jonas Frisén (KI) 

Application round: Feb 2014 

Subject: New Technology Development

Publications:

[ Ståhl _et al._ **Visualization and analysis of gene expression in tissue sections by spatial transcriptomics.** _Science_ 353:78-82 (2016) ](<http://dx.doi.org/10.1126/science.aaf2403>) [ Vickovic _et al._ **Massive and parallel expression profiling using microarrayed single-cell sequencing.** _Nat Commun_ 7:13182 (2016) ](<http://dx.doi.org/10.1038/ncomms13182>)

  * Spatially resolved gene expression analysis in human developing heart tissue  PI: Matthias Corbascio (KI) 

Application round: Oct 2016 

Subject: Cell and Developmental Biology

Publications:

[ Asp _et al._ **A spatiotemporal organ-wide gene expression and cell atlas of the developing human heart.** _Cell_ 179:1647-1660.e19 (2019) ](<http://dx.doi.org/10.1016/j.cell.2019.11.025>)

  * Speciation genomics in crows  PI: Jochen Wolf (UU) 

Application round: Jan 2013 

Subject: Evolutionary Biology/Genetics

Publications:

[ Poelstra _et al._ **The genomic landscape underlying phenotypic integrity in the face of gene flow in crows.** _Science_ 344:1410-1414 (2014) ](<http://dx.doi.org/10.1126/science.1253226>)

  * Stem cell mutational signatures  PI: Maria Eriksson (KI) 

Application round: Sep 2017 

Subject: Human Genetics

Publications:

[ Franco _et al._ **Whole genome DNA sequencing provides an atlas of somatic mutagenesis in healthy human cells and identifies a tumor-prone cell type.** _Genome Biol_ 20:285 (2019) ](<http://dx.doi.org/10.1186/s13059-019-1892-z>)

  * The Swedish reference database of genetic variants  PI: Ulf Gyllensten (UU) 

Application round: Oct 2015 

Subject: Human Genetics

Publications:

[ Ameur _et al._ **SweGen: a whole-genome data resource of genetic variability in a cross-section of the Swedish population.** _Eur J Hum Genet_ 25:1253-1260 (2017) ](<http://dx.doi.org/10.1038/ejhg.2017.130>) [ Ameur _et al._ **De novo assembly of two Swedish genomes reveals missing segments from the human GRCh38 reference and improves variant calling of population-scale sequencing data.** _Genes_ 9:486 (2018) ](<http://dx.doi.org/10.3390/genes9100486>)

Tools and scripts:

[ **Structural variation pipeline** – A workflow for standardized calling and filtering of structural variants, designed for whole genome sequencing projects ](<https://github.com/NBISweden/wgs-structvar>)

Data: [Genetic variant frequencies](<https://swefreq.nbis.se>)

  * The herring genome  PI: Leif Andersson (UU) 

Application round: Jan 2013 

Subject: Evolutionary Biology/Genetics

Publications:

[ Martinez Barrio _et al._ **The genetic basis for ecological adaptation of the Atlantic herring revealed by genome sequencing.** _Elife_ 5:e12081 (2016) ](<http://dx.doi.org/10.7554/eLife.12081>)

Data: [Genome assembly](<http://www.ncbi.nlm.nih.gov/assembly/318681>) [Gene annotation and genetic variants](<https://doi.org/10.5061/dryad.5r774>) [Population sequencing reads, set 1](<http://www.ncbi.nlm.nih.gov/sra/?term=SRP017094>) [Population sequencing reads, set 2](<http://www.ncbi.nlm.nih.gov/sra/?term=SRP017095>) [Population sequencing reads, set 3](<http://www.ncbi.nlm.nih.gov/sra/?term=SRP056617>)

  * The transcriptional heterogeneity of human intestinal innate lymphoid cells  PI: Jenny Mjösberg (KI) 

Application round: Sep 2017 

Subject: Cell and Developmental Biology

Publications:

[ Mazzurana _et al._ **Tissue-specific transcriptional imprinting and heterogeneity in human innate lymphoid cells revealed by full-length single-cell RNA-sequencing.** _Cell Res_ 31:554-568 (2021) ](<http://dx.doi.org/10.1038/s41422-020-00445-x>)

Tools and scripts:

[ **Analysis scripts** – to reproduce the analyses reported in the publication ](<https://github.com/czarnewski/ilc_tissue_imprinting>)

Data: [Gene expression data](<https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE150050>)

  * Transcriptomics of plant-insect coevolution  PI: Niklas Janz (SU) 

Application round: Oct 2013 

Subject: Evolutionary Biology/Genetics

Publications:

[ Celorio-Mancera _et al._ **Evolutionary history of host use, rather than plant phylogeny, determines gene expression in a generalist butterfly.** _BMC Evol Biol_ 16:59 (2016) ](<http://dx.doi.org/10.1186/s12862-016-0627-y>)

Tools and scripts:

[ **Gene expression analysis** – R and python scripts ](<https://github.com/hussius/butterfly>)

Data: [RNA-seq data](<https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-3861>)

  * Travel and antibiotic resistance  PI: Joakim Larsson (GU) 

Application round: Jan 2013 

Subject: Biodiversity

Publications:

[ Bengtsson-Palme _et al._ **The human gut microbiome as a transporter of antibiotic resistance genes between continents.** _Antimicrob Agents Chemother_ 59:6551-6560 (2015) ](<http://dx.doi.org/10.1128/AAC.00933-15>)

  * Tumorgan single cell sequencing  PI: Kristian Pietras (LU) 

Application round: Feb 2016 

Subject: Cancer Biology

Publications:

[ Bartoschek _et al._ **Spatially and functionally distinct subclasses of breast cancer-associated fibroblasts revealed by single cell RNA sequencing.** _Nat Commun_ 9:5150 (2018) ](<http://dx.doi.org/10.1038/s41467-018-07582-3>)

  * Whole genome sequencing of neuroblastoma  PI: Tommy Martinsson (GU) 

Application round: Oct 2015 

Subject: Cancer Biology

Tools and scripts:

[ **Sarek** – Complete open source pipeline to detect germline or somatic variants from normal or tumour/normal whole-genome sequencing data. ](<http://opensource.scilifelab.se/projects/sarek/>)

  * Zebrafish mutation identification  PI: Olov Andersson (KI) 

Application round: Feb 2014 

Subject: Cell and Developmental Biology

Publications:

[ Reischauer _et al._ **Cloche is a bHLH-PAS transcription factor that drives haemato-vascular specification.** _Nature_ 535:294-298 (2016) ](<http://dx.doi.org/10.1038/nature18614>)

Data: [Whole-genome sequencing of zebrafish on PacBio](<http://www.ncbi.nlm.nih.gov/sra/?term=PRJEB13442>) [Assembly of the Cloche/npas4l locus](<https://www.ncbi.nlm.nih.gov/nuccore/LT571435>)



