---
title:  'Tools'
visible: true
---
    

# Tools

The current tools under development and maintenance by NBIS staff are listed below. For more detailed information on the tools, follow the links. See also the [SciLifeLab Open Source website](<http://opensource.scilifelab.se>) for tools developed at other SciLifeLab platforms.

AGAT   
URL: <https://github.com/NBISweden/AGAT>   


Another Gff Analysis Toolkit. Suite of tools to handle gene annotations in any GTF/GFF format.

Software type: Command-line tool  
Platform: Linux, Mac, Windows  
Development status: Maintained  
Released: Yes  
Affiliation: nbis.se   


ChIP-seq QC pipeline   
URL: <https://github.com/ewels/clusterflow>   


Pipeline for quality control of ChIP-seq data, implemented as a plug-in for the workflow manager ClusterFlow. See pipelines/chipseq_qc.config in the ClusterFlow repository on GitHub.

Software type: Workflow  
Platform: Linux  
Development status: Maintained  
Released: Yes  
Contact: [jakub.westholm@scilifelab.se](<mailto:jakub.westholm@scilifelab.se>)  
Affiliation: nbis.se   


EMBLmyGFF3   
URL: <https://github.com/NBISweden/EMBLmyGFF3>   
Publications: [Norling, Jareborg & Dainat, 2018](<http://dx.doi.org/10.1186/s13104-018-3686-x>)

Software to convert GFF3 and fasta to legal EMBL format suitable for the submission to the European Nucleotide Archive (ENA) database.

Software type: Command-line tool  
Platform: Linux, Mac  
Development status: Maintained  
Released: Yes  
Affiliation: nbis.se   


GUESSmyLT   
URL: <https://github.com/NBISweden/GUESSmyLT>   
Publications: [Berner Wik, Olin, Vigetun Haughey, Klasson & Dainat, 2019](<http://dx.doi.org/10.21105/joss.01344>)

Software to guess the RNA-Seq library type of paired and single end read files using mapping and gene annotation.

Software type: Command-line tool  
Platform: Linux, Mac, Windows  
Development status: Maintained  
Released: Yes  
Affiliation: nbis.se   


IgDiscover   
URL: <http://igdiscover.se/>   
Publications: [Corcoran et al., 2016](<http://dx.doi.org/10.1038/ncomms13642>)

IgDiscover analyzes antibody repertoires and discovers new V genes from high-throughput sequencing reads. Heavy chains, kappa and lambda light chains are supported (to discover VH, VK and VL genes).

Software type: Command-line tool  
Platform: Linux, Mac  
Development status: Active  
Released: Yes  
Contact: <http://igdiscover.se/>  
PI: Gunilla Karlsson Hedestam  
Affiliation: ki.se   


Metabolic Atlas   
Publications: [Wang et al., 2021](<http://dx.doi.org/10.1073/pnas.2102344118>) , [Robinson et al., 2020](<http://dx.doi.org/10.1126/scisignal.aaz1482>)

Metabolic Atlas integrates open source genome-scale metabolic models (GEMs) of human and yeast for easy browsing and analysis.

Software type: Database portal, Web application, Web API, Web service  
Development status: Active  
Released: Yes  
Contact: [contact@metabolicatlas.org](<mailto:contact@metabolicatlas.org>)  
PI: Jens Nielsen  
Affiliation: chalmers.se   


MrBayes   
URL: <http://nbisweden.github.io/MrBayes/>   
Publications: [Huelsenbeck & Ronquist, 2001](<http://dx.doi.org/10.1093/bioinformatics/17.8.754>)

MrBayes is a program for Bayesian inference and model choice across a wide range of phylogenetic and evolutionary models. MrBayes uses Markov chain Monte Carlo (MCMC) methods to estimate the posterior distribution of model parameters.

Software type: Command-line tool  
Platform: Linux, Windows, Mac  
Development status: Maintained  
Released: Yes  
Contact: <http://nbisweden.github.io/MrBayes/>  
PI: Fredrik Ronquist  
Affiliation: nrm.se   


PRECOG   
URL: <http://precog.lundberg.gu.se/>   
Publications: [Fernandez-Ricaud et al., 2016](<http://dx.doi.org/10.1186/s12859-016-1134-2>)

PRECOG is a tool developed to automate visualization, analysis and exploration of complex and highly resolved microbial growth data, as well as standardized extraction of the fitness components it contains. We hope that this tool will promote simplicity, transparency and standardization in microbial growth phenomics, and provide a portal with the standalone software and online tools so that other microbiology labs can easily upload, assess and visualize their growth data.

Software type: Desktop application, Command-line tool, Web application  
Platform: Windows  
Development status: Maintained  
Released: Yes  
Contact: [luciano.fernandez@marine.gu.se](<mailto:luciano.fernandez@marine.gu.se>)  
PI: Anders Blomberg  
Affiliation: gu.se   


Proteios   
URL: <http://www.proteios.org>   
Publications: [Garden et al., 2005](<http://dx.doi.org/10.1093/bioinformatics/bti291>)

A multi-user platform for analysis and management of proteomics data.

Software type: Web application  
Development status: Maintained  
Released: Yes  
Contact: <http://ms-utils.org>  
PI: Fredrik Levander  
Affiliation: immun.lth.se   


Sarek   
URL: <https://nf-co.re/sarek>   
Publications: [Garcia M al. 2020](<http://dx.doi.org/10.12688/f1000research.16665.2>)

Sarek is a workflow designed to detect variants on whole genome or targeted sequencing data. Initially designed for Human, and Mouse, it can work on any species with a reference genome. Sarek can also handle tumour / normal pairs and could include additional relapses. Previously known as Cancer Analysis Workflow (CAW).

Software type: Command-line tool, Workflow  
Platform: Linux  
Development status: Active  
Released: Yes  
Contact: [maxime.garcia@scilifelab.se](<mailto:maxime.garcia@scilifelab.se>)  
Affiliation: scilifelab.se   


SweFreq   
URL: <https://swefreq.nbis.se>   
Publications: [Ameur et al., 2017](<http://dx.doi.org/10.1038/ejhg.2017.130>)

Server hosting whole-genome variant frequencies for 1000 Swedish individuals generated within the SweGen project.

Software type: Web application  
Development status: Maintained  
Released: Yes  
Contact: [swefreq@scilifelab.se](<mailto:swefreq@scilifelab.se>)  
PI: Ulf Gyllensten  
Affiliation: scilifelab.se   


The Human Protein Atlas   
URL: <http://www.proteinatlas.org/>   
Publications: [Uhlen et al., 2010](<http://dx.doi.org/10.1038/nbt1210-1248>) , [Uhlen et al., 2015](<http://dx.doi.org/10.1126/science.1260419>)

Database with millions of high-resolution images. Expression and localization of proteins in a large variety of normal human tissues, cancer cells and cell lines with the aid of immunohistochemistry (IHC) images and immunofluorescence (IF) confocal microscopy images.

Software type: Database portal  
Development status: Active  
Released: Yes  
Contact: [contact@proteinatlas.org](<mailto:contact@proteinatlas.org>)  
PI: Mathias Uhl√©n  
Affiliation: kth.se   
Used in: [Kampf et al., 2014](<https://dx.doi.org/10.1096/fj.14-250555>) , [Lindskog et al., 2015](<https://dx.doi.org/10.1186/s12864-015-1686-y>) , [Edfors et al., 2016](<https://dx.doi.org/10.15252/msb.20167144>)

Topcons   
URL: <https://topcons.net/>   
Publications: [Tsirigos et al., 2015](<http://dx.doi.org/10.1093/nar/gkv485>)

Consensus prediction of membrane protein topology and signal peptides

Software type: Web application  
Development status: Maintained  
Released: Yes  
Contact: <http://bioinfo.se/>  
PI: Arne Elofsson  
Affiliation: dbb.su.se   


wgs-structvar   
URL: <https://github.com/NBISweden/wgs-structvar>   
Publications: [Ameur et al., 2017](<http://dx.doi.org/10.1038/ejhg.2017.130>)

A workflow for standardized calling and filtering of structural variants, designed for whole genome sequencing projects.

Software type: Command-line tool  
Platform: Linux  
Development status: Maintained  
Released: Yes  
Contact: <https://nbis.se>  
PI:   
Affiliation: nbis.se   


WhatsHap   
URL: <http://whatshap.readthedocs.io>   
Publications: [Patterson et al., 2015](<http://dx.doi.org/10.1089/cmb.2014.0157>) , [Garg et al., 2016](<http://dx.doi.org/10.1101/037101>)

WhatsHap is a software for phasing genomic variants using DNA sequencing reads, also called haplotype assembly. It is especially suitable for long reads, but works also well with short reads.

Software type: Command-line tool  
Platform: Linux, Mac  
Development status: Active  
Released: Yes  
Contact: <https://lists.cwi.nl/mailman/listinfo/whatshap>  
PI: Tobias Marschall  
Affiliation: mpi-inf.mpg.de   


Xferticket   
URL: <https://xferticket.nbis.se>   


Service for facilitating transfer of big datasets to and from nbis experts.

Software type: Web application  
Development status: Maintained  
Released: Yes  
Contact: [cto@nbis.se](<mailto:cto@nbis.se>)  
Affiliation: nbis.se   


contigtax UNDER DEVELOPMENT   
URL: <https://github.com/NBISweden/contigtax>   


contigtax assigns taxonomy to contigs in metagenomic assemblies using sequence alignments against protein databases (e.g. UniRef) and calculation of lowest common ancestor in the taxonomic hierarchy.

Software type: Command-line tool  
Platform: Linux, Mac  
Development status: Active  
Released: No  
Contact: [john.sundh@nbis.se](<mailto:john.sundh@nbis.se>)  
Affiliation: nbis.se   


HaloPlex variant calling pipeline UNDER DEVELOPMENT   
Publications: [Tuominen et al., 2016](<http://dx.doi.org/10.1002/gcc.22363>) , [Dalin et al., 2017](<http://dx.doi.org/10.1016/j.ijcard.2016.11.066>)

A pipeline for calling variants from targeted DNA sequencing data obtained using the HaloPlex method.

Software type: Workflow  
Platform: Linux  
Development status: Active  
Released: No  
Contact: [par.engstrom@scilifelab.se](<mailto:par.engstrom@scilifelab.se>)  
Affiliation: nbis.se   


LocalEGA UNDER DEVELOPMENT   
URL: <https://github.com/NBISweden/LocalEGA>   


The European Genome-phenome Archive (EGA) is a service for permanent archiving and sharing of all types of personally identifiable genetic and phenotypic data resulting from biomedical research projects. Local EGA handles data that are supposed to stay in Sweden.

Software type: Command-line tool, Workflow  
Platform: Linux, Mac  
Development status: Active  
Released: No  
Contact: [ega@nbis.se](<mailto:ega@nbis.se>)  
Affiliation: nbis.se   


Single-cell RNA-seq analysis workflow UNDER DEVELOPMENT   
URL: <https://bitbucket.org/scilifelab-lts/lts-workflows-sm-scrnaseq/branch/develop>   


A workflow for quality control and basic analysis of single-cell RNA-seq data.

Software type: Workflow  
Platform: Linux  
Development status: Active  
Released: No  
Contact: [asa.bjorklund@scilifelab.se](<mailto:asa.bjorklund@scilifelab.se>)  
Affiliation: nbis.se   

